tests/e2e/performance/csi_tests/test_pvc_attachtime.py

```diff
@@ -237,9 +241,26 @@ def test_pod_start_time(self, pod_obj_list):
         self.full_results.add_key("pvc_size", self.pvc_size)
 
         # Write the test results into the ES server
-        self.full_results.es_write()
+        if self.full_results.es_write():
+            res_link = self.full_results.results_link()
+            log.info(f"The Result can be found at : {res_link}")
 
-        # write the ES link to the test results in the test log.
-        log.info(
-            f"{self.msg_prefix} The Result can be found at : {self.full_results.results_link()}"
+            # Create text file with results of all subtest (4 - according to the parameters)
+            self.write_result_to_file(res_link)
+
+    def test_pod_start_time_results(self):
+        """
+        This is not a test - it is only check that previous test ran and finish as expected
+        and reporting the full results (links in the ES) of previous tests (2)
+        """
+
+        self.number_of_tests = 2
+        self.results_path = get_full_test_logs_path(
+            cname=self, fname="test_pod_start_time"
         )
+        self.results_file = os.path.join(self.results_path, "all_results.txt")
+        log.info(f"Check results in {self.results_file}")
+
+        self.check_tests_results()
+
+        self.push_to_dashboard(test_name=self.benchmark_name)
```

Comment:
benchmark name is : `pvc_attach_time`
i think we want in the dashboard something like `PVC Attach Time`, WDYT ?

Reply:
> 

PVC attach time sounds good. I should not change anything in the code, it's the name in the dashboard only, right ?

Reply:
you need to change in this line from `test_name=self.benchmark_name` to `test_name="PVC Attach Time"`
so in the dashboard it will display as : `PVC Attach Time`

Reply:
Done